hudi-client/src/test/java/org/apache/hudi/callback/http/TestHoodieWriteCommitHttpCallback.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.callback.http;
+
+import org.apache.hudi.callback.impl.HoodieWriteCommitHttpCallback;
+import org.apache.hudi.testutils.HoodieWriteCommitTestHarness;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+/**
+ * Unit test for {@link HoodieWriteCommitHttpCallback}.
+ */
+public class TestHoodieWriteCommitHttpCallback extends HoodieWriteCommitTestHarness {
```

Comment:
What's purpose of this class?

Reply:
> What's purpose of this class?

its purpose is to test the Initialization process of `HoodieWriteCommitHttpCallback` when provided with proper params referring to this test 'org.apache.hudi.metrics.datadog.TestDatadogMetricsReporter#instantiationShouldSucceed'.

this test might be unnecessary，I'll remove it if you think it is unnecessary too

Reply:
IMO, it's not necessary. We can remove it.

Reply:
done